setup.py

```diff
@@ -46,8 +46,10 @@
       scripts=glob('scripts/*'),
       extras_require={'test': ["nose >= 0.10.1", "pep8"],
                       'doc': ["Sphinx >= 1.2.2", "sphinx-bootstrap-theme"]},
-      install_requires=['click', 'numpy >= 1.7',
+      install_requires=['click >= 6', 'numpy >= 1.7',
                         'scikit-bio >= 0.2.2, < 0.3.0',
                         'biom-format >= 2.1.3, < 2.2.0'],
+      dependency_links=[('https://github.com/pallets/click/archive/master.zip'
```

Comment:
Removed the click 7 dependency - we don't use 'hidden' now (and don't want to depend on dev version). i think it should work with click 6 (didn't test 5)
Don't know what the 'dependecy_links' means or how to fix it...